client/pom.xml

```diff
@@ -378,11 +378,6 @@
             <artifactId>gson</artifactId>
             <version>${gson-version}</version>
         </dependency>
-        <dependency>
```

Comment:
Sorry, I should've caught this in my last review - this file is generated from `client/templates/libraries/okhttp-gson/pom.mustache`, so these dependencies should be removed/updated there, and then the client re-gen'd.

I think this change to a gen'd file was able to survive a re-gen because the OpenAPI generator refuses to overwrite some files: 
```
./bin/refresh_client
...
[main] INFO  o.o.codegen.DefaultCodegen - Skipped overwriting pom.xml as the file already exists in /out//pom.xml
[main] INFO  o.o.codegen.DefaultCodegen - Skipped overwriting README.md as the file already exists in /out//README.md
```

I think our strategy in `bin/refresh_client` should include removing these files, and then generating the client lib.